documentation/asciidoc/accessories/camera/libcamera_options_common.adoc

```diff
@@ -19,6 +19,23 @@ libcamera-apps build: ca559f46a97a 27-09-2021 (14:10:24)
 libcamera build: v0.0.0+3058-c29143f7
 ----
 
+----
+	--config,	-c		Read options from the given file <filename>
+----
+
+Normally options are read from the command line, but in case multiple options are required it may be more convenient to keep them in a file.
+
+Example: `libcamera-hello -c config.txt`
```

Comment:
I wonder if `config.txt` is a bad-choice here, given the filename-overlap with https://www.raspberrypi.com/documentation/computers/config_txt.html :wink: 
Perhaps `camera-options.txt` or something?

Reply:
Slightly inclined to leave it as `config.txt`, the option is `--config` after all. I'm hoping our users have learnt to cope with the proliferation of "configurations" everywhere...!

Reply:
ping @aallan - do you think this is something that users might get confused by?

Reply:
I think that's okay, it's a fairly generic filename. I mean, that's not to say it isn't going to confuse *someone*. Everything always confuses *someone*. 🤷